src/test/java/org/prebid/cache/handlers/PostCacheHandlerTests.java

```diff
@@ -112,6 +113,8 @@ void testSecondaryCacheSuccess() throws InterruptedException {
         serverMock.stubFor(post(urlPathEqualTo("/cache"))
                 .willReturn(aResponse().withBody("{\"responses\":[{\"uuid\":\"f31f96db-8c36-4d44-94dc-ad2d1a1d84d9\"}]}")));
 
+        PostCacheHandler handler = new PostCacheHandler(repository, cacheConfig, metricsRecorder, builder, currentDateProvider);
```

Comment:
You are using mixed approach in variables naming, below in this test val is used, so replace types with val or val with types.